src/Dependencies/CodeAnalysis.Metadata/PortableCustomDebugInfoKinds.cs

```diff
@@ -9,6 +9,7 @@ internal static class PortableCustomDebugInfoKinds
         public static readonly Guid AsyncMethodSteppingInformationBlob = new Guid("54FD2AC5-E925-401A-9C2A-F94F171072F8");
         public static readonly Guid StateMachineHoistedLocalScopes = new Guid("6DA9A61E-F8C7-4874-BE62-68BC5630DF71");
         public static readonly Guid DynamicLocalVariables = new Guid("83C563C4-B4F3-47D5-B824-BA5441477EA8");
+        public static readonly Guid TupleLocalVariables = new Guid("ED9FDF71-8879-4747-8ED3-FE5EDE3CE710");
```

Comment:
>TupleLocalVariables [](start = 36, length = 19)

Should this be called `TupleElementNames` for consistency with `CustomDebugInfoKind` enum?